data/seeds/01_users.js

```diff
@@ -0,0 +1,12 @@
+
+exports.seed = function(knex, Promise) {
+  return knex('Users').truncate()
+    .then(function () {
+      return knex('Users').insert([
+        {Username: "UserOne", Password: "1234567zyxw"},
```

Comment:
Do not suggest seeding passwords or in this case user accounts, because this information is not secure without hashing